inc/Engine/Admin/Settings/Settings.php

```diff
@@ -350,6 +376,18 @@ public function sanitize_callback( $input ) {
 			$input['sitemaps'] = [];
 		}
 
+		// Option : fonts to preload.
+		if ( ! empty( $input['preload_fonts'] ) ) {
+			if ( ! is_array( $input['preload_fonts'] ) ) {
+				$input['preload_fonts'] = explode( "\n", $input['preload_fonts'] );
+			}
+			$input['preload_fonts'] = array_map( [ $this, 'sanitize_font' ], $input['preload_fonts'] );
+			$input['preload_fonts'] = array_filter( $input['preload_fonts'] );
+			$input['preload_fonts'] = array_unique( $input['preload_fonts'] );
```

Comment:
I think it would be better if this part is in the `sanitize_font` callback, then we can do a ternary instead:

`$input['preload_fonts'] = ! empty( $input['preload_fonts'] ) ? $this->sanitize_font( $input['preload_fonts'] ) : [];`

Reply:
Done in https://github.com/wp-media/wp-rocket/pull/2555/commits/96953434ee3007a2a22f1127a3daae1bb4e1dc45.